src/include/distributed/multi_physical_planner.h

```diff
@@ -274,6 +281,13 @@ typedef struct Task
 	 * the task splitted into local and remote tasks.
 	 */
 	bool partiallyLocalOrRemote;
+
+	/*
+	 * When we evaluate functions and parameters in the query tree then
+	 * we should no longer send the list of parameters long with the
+	 * query.
+	 */
+	bool parametersRemovedFromQueryString;
```

Comment:
> Would it complicate things if we try to fetch it from Job whenever needed in Task?

Yes, because the adaptive executor does not know about the Job. It is not part of the DistributedExecution.